spec/helpers/your_details_form_helper_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe Wpcc::YourDetailsFormHelper, type: :helper do
+  describe '.disable_minimum_contribution_option?' do
+    subject { helper.disable_minimum_contribution_option? }
+
+    context 'salary lower than threshold' do
+      let(:form) do
+        Wpcc::YourDetailsForm.new(age: age,
+                                  gender: gender,
+                                  salary: salary,
+                                  salary_frequency: salary_frequency,
+                                  contribution_preference: pref)
+      end
+
+      before do
+        form.valid?
+        assign(:your_details_form, form)
+      end
+
+      context 'make minimum contribution preference' do
+        let(:age)    { 35 }
+        let(:gender) { 'female' }
+        let(:salary) { 4000 }
+        let(:pref)   { 'minimum' }
+        let(:salary_frequency) { 'year' }
```

Comment:
I am ok with that solution @Tr4pSt3R.
I am happy for this to be a case by case situation - where in some cases, where it isn't going to break the 80 limit we can inline the `{`
My larger point was, if we are doing that, then all of them should be inlined for that 'block'